pset-functions.js

```diff
@@ -136,6 +171,86 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function scrabbleScore(word) {
+  const letters = word.split('')
+  let counter = 0
+  for (let i = 0; i <= word.length; i++) {
+    if (letters[i] === 'a' || letters[i] === 'e' || letters[i] === 'i' || letters[i] === 'o' || letters[i] === 'u' || letters[i] === 'l' || letters[i] === 'n' || letters[i] === 'r' || letters[i] === 's' || letters[i] === 't') {
```

Comment:
@anniecdavis27 nice set up. Also, I like your use of the a logical or `||` to map group the letters to their score